frontend/src/components/InterestsDropdown.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import InterestsDropdown from './InterestsDropdown';
+import renderer from 'react-test-renderer';
+
+describe('Interests Dropdown', () => {
+  it('should be defined', () => {
+    expect(InterestsDropdown).toBeDefined();
+  });
+  it('should render correctly', () => {
+    const tree = renderer.create(<InterestsDropdown />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
it should be populated with a list of interests

it should be populated in alphabetical order

Reply:
Thank you for the suggestion, changed!

Reply:
It looks like the first suggested test didn't make it in. The alphabetical order test won't prevent somebody from changing the dropdown order to non-alphabetic order as long as the new snapshot is approved. I would expect this test to grab the list of interests from the dropdown and assert that they're already sorted.

Reply:
Added both tests, thanks!